_episodes/03-changes.md

```diff
@@ -407,23 +425,23 @@ $ git log
 {: .bash}
 
 ~~~
-commit c769a0d20bf6b3060cd3c752f4def937880fa4eb (HEAD -> master)
+commit fefd5552f740e814dbb40f9247eddadf2dec4c82 (HEAD -> master)
 Author: Catsy Cline <ccline@pawson.edu>
-Date:   Wed Oct 25 15:57:06 2017 -0700
+Date:   Thu Nov 2 12:10:40 2017 -0700
 
     add pictures of cats
 
-commit 06156ef7588b1359b467bd277b290f04b03fd094
+commit 939b6f04023d267e4036b269dbcbb45cb1638a1b
 Author: Catsy Cline <ccline@pawson.edu>
-Date:   Wed Oct 25 15:47:58 2017 -0700
+Date:   Thu Nov 2 12:07:41 2017 -0700
 
     readme: where are the files?
 
-commit 73f865ac9f0b3e790328d11ec6cd7653208a5a3e
+commit 9ab6e19c9872a23674772ea5a514f05a65ba5d9e
 Author: Catsy Cline <ccline@pawson.edu>
-Date:   Wed Oct 25 15:44:38 2017 -0700
+Date:   Thu Nov 2 12:03:47 2017 -0700
 
-    add notes on metadata
+    add notes on metadata, and the metadata
```

Comment:
might want to check the rest of the episodes for this commit message... i think it is used elsewhere as an example